bot/exts/utilities/bookmark.py

```diff
@@ -25,6 +23,49 @@
 )
 
 
+class LinkTargetMessage(discord.ui.View):
+    """The button that relays the user to the bookmarked message."""
+
+    def __init__(
+        self,
+        target_message: discord.Message,
+    ):
+        super().__init__()
+        self.add_item(discord.ui.Button(label="View Message", url=target_message.jump_url))
+
+
+class SendBookmark(discord.ui.View):
+    """The button that sends the bookmark to other users."""
+
+    def __init__(
+        self,
+        bookmark_function: Callable[[discord.TextChannel, discord.Member, discord.Message, str], None],
+        author: discord.Member,
+        channel: discord.TextChannel,
+        target_message: discord.Message,
+    ):
+        super().__init__()
+
+        self.bookmark_function = bookmark_function  # The function is the 'action_bookmark'
+        self.clicked: list[int] = [author.id]
+        self.channel = channel
+        self.target_message = target_message
+
+    @discord.ui.button(label="Receive Bookmark", style=discord.ButtonStyle.green)
+    async def button_callback(self, button: discord.Button, interaction: discord.Interaction) -> None:
+        """The button callback."""
+        if interaction.user.id in self.clicked:
+            return await interaction.response.send_message(
+                "You have already received a bookmark to that message.", ephemeral=True
+            )
```

Comment:
```suggestion
            await interaction.response.send_message(
                "You have already received a bookmark to that message.",
                ephemeral=True,
            )
            return
```
To follow the type hint of None.